reasoner/resolution/resolver/ConjunctionResolver.java

```diff
@@ -206,6 +218,33 @@ protected REQ_STATE requestStateReiterate(Request fromUpstream, REQ_STATE reques
         }
     }
 
+
+    public static class RequestState extends CompoundResolver.RequestState {
```

Comment:
this behaviour is now in all conjunction resolvers, so we only have 1 type of Conjunction request state